cursorless-talon/src/cursorless.talon

```diff
@@ -24,3 +24,8 @@ cursorless instructions:   user.cursorless_open_instructions()
 cursorless docks:          user.cursorless_open_instructions()
 cursorless reference:      user.cursorless_cheat_sheet_show_html()
 cursorless cheat sheet:    user.cursorless_cheat_sheet_show_html()
+cursorless settings:
```

Comment:
One concern is that we also have Talon-side configuration, eg spoken forms, so I'm slightly hesitant for this voice command to mean VSCode settings

Reply:
We could of course open a little imgui with buttons/voice commands that will open a csv file or vscode settings

Reply:
Yeah that's fair; we could always do that later, so prob fine to ship it like this for now